### PR TITLE
[16.0][IMP] edi_oca: add smart button in record form to show all related queue jobs

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

File: edi_oca/models/edi_exchange_record.py [hunk 1]
```diff
@@ -603,3 +607,26 @@ def delayable(self, **kw):
...
+    def action_view_related_queue_jobs(self):
+        self.ensure_one()
+        xmlid = "queue_job.action_queue_job"
+        action = self.env["ir.actions.act_window"]._for_xml_id(xmlid)
+        # Searching based on task name. Ex: `edi.exchange.record(1,).action_exchange_send()`
+        action["domain"] = [("func_string", "like", str(self))]
+        # Purge default search filters from ctx to avoid hiding records
+        ctx = action.get("context", {})
+        if isinstance(ctx, str):
+            ctx = safe_eval.safe_eval(ctx, self.env.context)
```
Comment (nilshamerlinck) [inline edi_oca/models/edi_exchange_record.py]: safe_eval a bit overkill here, literal_eval not enough?

**Comment by QuocDuong1306** [comment]
Thank @nilshamerlinck , I updated the PR

File: edi_oca/views/edi_exchange_record_views.xml [hunk 2]
```diff
@@ -88,6 +88,22 @@
...
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-calendar"
+                            name="action_view_related_queue_jobs"
+                            attrs="{'invisible': [('related_queue_jobs_count', '=', 0)]}"
+                        >
+                            <field
+                                string="Queue(s)"
```
Comment (nilshamerlinck) [inline edi_oca/views/edi_exchange_record_views.xml]:
```suggestion
                                string="Jobs"
```

**Review by nilshamerlinck — APPROVED** [review]
pre-approving

**Pull request closed without merging**